packages/components/src/index.js

```diff
@@ -59,3 +60,4 @@ export { default as TextControlWithAffixes } from './text-control-with-affixes';
 export { default as useFilters } from './higher-order/use-filters';
 export { default as ViewMoreList } from './view-more-list';
 export { default as WebPreview } from './web-preview';
+export { SlotFillProvider } from '@wordpress/components';
```

Comment:
Why does this need to be exported? Is it minimum version requirement for WP thing?

Reply:
Oh, I see now. So we can guarantee its available for extensions.